app/models/category.rb

```diff
@@ -0,0 +1,8 @@
+class Category < ApplicationRecord
+  acts_as_paranoid
+
+  has_many :product, dependent: :destroy
```

Comment:
quan hệ nhiều thì thêm số nhiều vào em, fix tương tự bên dưới
```suggestion
  has_many :products, dependent: :destroy
```